tests/handlebars/mustache-statement/basics.hbs

```diff
@@ -28,15 +28,32 @@
 <div
   class="a-first-class
     {{if this.optionOne "optionOne"}}
-    
+
```

Comment:
This change could be reverted

Reply:
I'll revert it at the end - It caused by my editor, thanks for the catch.

Reply:
Done